src/lib/utilities/request-from-api.ts

```diff
@@ -55,9 +58,8 @@ export const requestFromAPI = async <T>(
   });
 
   const url = toURL(base + '/api/v1' + encode(endpoint), query);
```

Comment:
Since we're attaching the query on here, I'm not sure if you _also_ need to put them back on in Line 62.